docs/course_materials/linux_intro.html

```diff
@@ -0,0 +1,367 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+
+  <title>Linux Introduction</title>
+
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+  <link rel="stylesheet" href="reveal/css/reveal.css">
+  <link rel="stylesheet" href="media/presentations.css" id="theme">
+  <link rel="stylesheet" href="reveal/lib/css/zenburn.css">
+
+  <!-- Printing and PDF exports -->
+  <script>
+    var link = document.createElement('link');
+    link.rel = 'stylesheet';
+    link.type = 'text/css';
+    link.href = window.location.search.match(/print-pdf/gi) ? 'reveal/css/print/pdf.css' : 'reveal/css/print/paper.css';
+    document.getElementsByTagName('head')[0].appendChild(link);
+  </script>
+
+  <style>
+    .reveal .slide spre {
+      width: 100%
+    }
+  </style>
+</head>
+
+<body>
+
+  <div class="reveal">
+    <div class="slides">
+
+      <section data-background-size="cover" data-background-image="media/rose_background.jpg">
+        <h1 class="slide_main_title"><b>Linux Introduction</b></h1>
+      </section>
+      <section>
+        <section>
+          <h3><b>Open a terminal window</b></h3>
+          <ul>
+            <li>Open the search menu:</li>
+            <ul>
+              <li>Press the "Windows" key</li>
+              <li>Or move the mouse to the upper left of the screen</li>
+            </ul>
+            <li>Type "terminal" in the search bar</li>
+            <li>Click on the icon named "Terminal"</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$
+            </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>Getting more info on commands</b></h3>
```

Comment:
Maybe we should move this one for later, cause they didn't learn any command yet :)

Reply:
Done, moved to the end